core/cmd/remote_client.go

```diff
@@ -548,3 +555,39 @@ func parseResponse(resp *http.Response) ([]byte, error) {
 	}
 	return b, err
 }
+
+func (cli *Client) checkRemoteBuildCompatibility(lggr logger.Logger, c *clipkg.Context) error {
+	resp, err := cli.HTTP.Get("/v2/build_info")
+	if err != nil {
+		lggr.Warn("Remote node version is unknown. CLI may behave in unexpected ways.")
```

Comment:
I would suggest logging the error here. Like so:

```suggestion
		lggr.Warnw("Got error querying for version, remote node version is unknown. CLI may behave in unexpected ways.", "err", err)
```